bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/SensorThingHandler.java

```diff
@@ -333,9 +329,30 @@ public void initialize() {
         }
     }
 
-    public void valueUpdated(ChannelUID channelUID, SensorState newState, boolean initializing) {
-        this.state = newState;
+    public void valueUpdated(ChannelUID channelUID, SensorConfig newConfig) {
+        Integer batteryLevel = newConfig.battery;
+        Float temperature = newConfig.temperature;
+
+        switch (channelUID.getId()) {
+            case CHANNEL_BATTERY_LEVEL:
+                if (batteryLevel != null) {
+                    updateState(channelUID, new DecimalType(batteryLevel.longValue()));
+                }
+                break;
+            case CHANNEL_BATTERY_LOW:
+                if (batteryLevel != null) {
+                    updateState(channelUID, batteryLevel <= 10 ? OnOffType.ON : OnOffType.OFF);
```

Comment:
Not very vital, but you can also write:
```suggestion
                    updateState(channelUID, OnOffType.from(batteryLevel <= 10));
```

Reply:
Yes, that is possible.

tbh Currently I am not very happy with this hard-coded comparison. I guess it is done in other bindings too. But maybe a user wants battery low warnings depending on a different threshold? A config parameter seem to be to clumsy.

I thought about implementing a profile for this use case.

Reply:
It can also be implemented with a simple rule. Creating a profile seems to me as the profile is 'misused' as a method to create shortcuts for simple rules. I'm not sure if that is the intention of profiles? I can see a case for being able to easily create simple rule like features. And could be something for a rule template.

Reply:
I think you are right. A rule is sufficient.